Frends.MicrosoftSQL.ExecuteProcedure/README.md

```diff
@@ -0,0 +1,30 @@
+# Frends.MicrosoftSQL.ExecuteProcedure
+Frends MicrosoftSQL Task to execute MSSQL procedure.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions/workflows/ExecuteProcedure_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.MicrosoftSQL.ExecuteProcedure)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.MicrosoftSQL/Frends.MicrosoftSQL.ExecuteProcedure|main)
+
+# Installing
+
+You can install the Task via Frends UI Task View or you can find the NuGet package from the following NuGet feed https://www.myget.org/F/frends-tasks/api/v2.
+
+## Building
+
+
+Rebuild the project
+
+`dotnet build`
+
+Run tests
+
+ Create a simple SQL server to docker:
+ `docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Salakala123!" -p 1433:1433 --name sql1 --hostname sql1 -d mcr.microsoft.com/mssql/server:2019-CU18-ubuntu-20.04`
```

Comment:
Can we actually wrap this into a docker compose file and use that instead? Otherwise this is code duplication (as this code is mentioned in the GitHub actions and here too). It would also help development to just know to do `docker-compose up`, instead of every time copy-pasting this command line to terminal.